CONTRIBUTING.md

```diff
@@ -49,21 +49,16 @@ Merging Pull Requests (for project contributors with write access)
 - Install [Maven](http://maven.apache.org/download.cgi)
   - add `MAVEN_HOME` to environment variables
 
-
->**Note:** If you are on `Windows`, enable paths longer than 260 characters by: <br><br>
-1.- Run this as Administrator on a command prompt:<br> 
-`REG ADD HKLM\SYSTEM\CurrentControlSet\Control\FileSystem /v LongPathsEnabled /t REG_DWORD /d 1`<br>*(might need to type `yes` to override key if it already exists)*<br><br>
-2.- Set up `git` by running:<br> `git config --system core.longpaths true`
-
 ### Building and Testing
 
 The easiest way to build is by running the following command from the root folder:
```

Comment:
A command should be added that we need to build:
`mvn install -f eng\code-quality-reports\pom.xml` before running this install.

Reply:
I added -Dincluding-non-shipping-modules. This will build sdk-build-tool. Is there any special reason to have `mvn install -f eng\code-quality-reports\pom.xml`?

Reply:
Can someone outside of Microsoft successfully run the sdk-build-tools install command? It's unclear.

Reply:
@JimSuplizio 
We have non shipping module which includes sdk-build-tools build and install. 
I tried in local. It can pass if we simply add `-Dincluding-non-shipping-modules` into the origin command line. So there is no need to run `mvn install -f eng\code-quality-reports\pom.xml` before but simply one command line to build the project. 
`